wso2.org.component.javaparser/src/resources/ApplicationManagementServiceComponent.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
test files need to go to src/test/resources/

Reply:
added to src/test/resources/